app/config.py

```diff
@@ -22,6 +29,38 @@
 
 TIME_TO_CANCEL_ACTIONS = 60
 
+DEFAULT_DURATION = timedelta(hours=1)  # длительность ограничения по умолчанию
+FOREVER_DURATION = timedelta(days=666)  # длительность ограничения "навсегда"
+
+# auto restrict when karma less than NEGATIVE_KARMA_TO_RESTRICT
+ENABLE_AUTO_RESTRICT_ON_NEGATIVE_KARMA = bool(int(os.getenv("ENABLE_AUTO_RESTRICT_ON_NEGATIVE_KARMA", default=0)))
+
+NEGATIVE_KARMA_TO_RESTRICT = -100
+KARMA_AFTER_RESTRICT = -80
+
+
+class RestrictionPlanElem(typing.NamedTuple):
+    duration: timedelta
+    type_restriction: TypeRestriction
+
+
+RESTRICTIONS_PLAN: typing.List[RestrictionPlanElem] = [
```

Comment:
I have mixed feelings here.
Firstly, this seems to have no place in the config.
Secondly, it is not very clear, further you will see references to elements, for example `[-1]`
Maybe dict here?

Reply:
i chose between NamedTuple and dataclass and chose NamedTuple for its immutable. I dont like use dicts for fixed counts keys. strings-keys are not checked by linter, and it can't help me in case of typo